Document/0x05a-Platform-Overview.md

```diff
@@ -367,12 +367,39 @@ An example Broadcast Receiver declaration with an intent filter in a manifest:
 </receiver>
 ```
 
+The other way is to create the receiver dynamically in code and register it with the `Context.registerReceiver()` method.
+

```

Comment:
I know it is a little bit out of the way of your PR, but can I ask you a favor? Can you link to https://github.com/OWASP/owasp-mstg/blob/master/Document/0x05h-Testing-Platform-Interaction.md#broadcast-permission-enforcement to show how security matters and update that section with the security considerations of https://developer.android.com/guide/components/broadcasts#security-and-best-practices ?